e2e/test/scenarios/permissions/sandboxing/helpers/e2e-sandboxing-helpers.ts

```diff
@@ -337,35 +403,43 @@ export const valuesShouldContainOnlyGizmos = (
   expect(values).to.deep.equal(["Gizmo"]);
 };
 
-export const getDashcardResponses = (items: SandboxableItems) => {
+export const getDashcardResponses = (
+  dashboard: Dashboard | null,
+  questions: CollectionItem[],
+) => {
   signInAsNormalUser();
 
-  H.visitDashboard(items.dashboard.id);
+  H.visitDashboard(checkNotNull(dashboard).id);
```

Comment:
I don't care as much in tests, but I don't love checks like these when they're just to keep TS happy. Why can dashboard be null in this case?

Reply:
Just because it's initially defined as `null`, then populated in a `before` callback, before being used in the individual tests.